scipy/sparse/csgraph/tests/test_shortest_path.py

```diff
@@ -279,6 +287,12 @@ def check(method, directed):
             check(method, directed)
 
 
+@pytest.mark.parametrize("method", ['FW', 'J', 'BF'])
+def test_negative_weights(method):
+    SP = shortest_path(directed_negative_weighted_G, method, directed=True)
+    assert_array_almost_equal(SP, directed_negative_weighted_SP)
```

Comment:
minor: we should use `assert_allclose` for *new* testing code per docstring of https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_almost_equal.html

Reply:
done.